elliott/elliottlib/cli/get_golang_versions_cli.py

```diff
@@ -85,7 +86,7 @@ async def get_golang_versions_cli(runtime, release, advisory_id, default_advisor
         nvrs = [n.strip() for n in nvrs.split(',')]
         return print_nvrs_golang(nvrs, output, report)
     else:
-        util.red_print('The input value is not valid.')
+        format_util.red_print('The input value is not valid.')
```

Comment:
Since `green_print` is directly, imported, `red_print` can be imported as well right, and `format_util` removed from `from artcommonlib import rhcos, format_util` ?